### ISSUE TITLE
Add general interfaces to other services

### ISSUE THREAD

**Issue by dbosk** [oscarb/flowlist#2]
One thing I usually lack with services like this is to connect them. Usually I read stuff (RSS-feeds and email) in the terminal. Then I send stuff to my Kindle and use my email. So it would be nice to be able to forward an email to a Flowlist address, e.g. oscarb@[flowli.st](http://www.nic.st/register/step1/flowli), to add it to my list.

**Comment by oscarb** [comment]
Good idea! I will consider implementing an API at first to allow users to send links to their Flowlist.
